tree/ntuple/v7/src/RPageStorageFile.cxx

```diff
@@ -224,7 +224,11 @@ ROOT::Experimental::Detail::RPageSourceFile::~RPageSourceFile()
 ROOT::Experimental::RNTupleDescriptor ROOT::Experimental::Detail::RPageSourceFile::AttachImpl()
 {
    RNTupleDescriptorBuilder descBuilder;
-   const auto fNTuple = fReader.GetNTuple(fNTupleName);
+   auto ntplRes = fReader.GetNTuple(fNTupleName);
+   if (!ntplRes) {
+      ntplRes.Throw();
+   }
+   const auto fNTuple = ntplRes.Get();
```

Comment:
Along the way, we should also rename this to `ntpl` (or something else that does not start with `f`, which looks like a class member).

Reply:
Thank you, done in f46c020735e49f21c26fd7269a320ef835c4b64a